lib/guardian/validation.ts

```diff
@@ -0,0 +1,86 @@
+import { object, bool, string } from 'yup';
+/**
+ * Entire Guardian/POA validation schema
+ */
+
+export const guardianInformationSchema = object({
+  omitGuardianPoa: bool().default(false),
+  firstName: string()
+    .nullable()
+    .default(null)
+    .when('omitGuardianPoa', {
+      is: false,
+      then: string()
+        .typeError('Please enter a first name')
+        .matches(/^[a-zA-Z]*$/, 'Must only contain letters')
```

Comment:
I think we can remove the matches line. For instance, this won't allow names with hyphens. Even though other non-alpha characters are pretty unlikely to be in names I think it's okay to allow it to be any string which is how we're handling this in other forms. Same goes for middle name, last name, and city.

Reply:
Good point, I didn't think about the hyphens :)